### DISCUSSION TITLE
What would you like in a Muzei redesign?

### DISCUSSION THREAD

**Discussion by ianhanniballake** [romannurik/muzei#707] [General]
@romannurik posted some [design explorations](https://twitter.com/romannurik/status/1344383269480116231) of what a Muzei redesign might look like.

What parts of that redesign were really exciting for you? What would you like to see?

**Comment by ianhanniballake** [comment]
Personally, I really, really liked the redesign of the Effects screen. The 'Home', Locked icon, and 'Lock' really made it way easier to see when both were linked together vs independent from one another.

**Comment by romannurik** [comment]
One of my favorite parts to work on was the source card redesign. The design relies on a circle mask for source icons (and not all devices/users use circle icons), but I figure either (a) we'd force the icons into circles or (b) have a simpler UI when the system uses non-circles, assuming you could detect the system icon shape. Beyond that, the consistency in layout regardless of a source's having or artwork or not might be nice too.
